### PR TITLE
Refactor msbuild props

### PR DESCRIPTION
**Pull request by rstm-sf**
Refactor msbuild props. This will make it easier to keep track of the language version and nullability.

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<Project>
+
+    <PropertyGroup Condition=" '$(MSBuildProjectExtension)' == '.csproj' ">
+        <LangVersion>8.0</LangVersion>
+        <IsEnableNullable Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '3.0')) ">True</IsEnableNullable>
+        <IsEnableNullable Condition=" '$(TargetFrameworkIdentifier)' == '.NETStandard' AND $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '2.1')) ">True</IsEnableNullable>
+        <Nullable Condition=" '$(IsEnableNullable)' == 'True' ">enable</Nullable>
+        <Nullable Condition=" '$(IsEnableNullable)' != 'True' ">annotations</Nullable>
+    </PropertyGroup>
```
Comment (ForNeVeR) [inline Directory.Build.props:9]: I'm sorry, but this doesn't seem to really work.

Here's how I'm checking it:
1. Change `Atom WithPreviousAtom(DummyAtom? previousAtom);` with `Atom WithPreviousAtom(DummyAtom previousAtom);` in `IRow.cs`
2. `cd src/WpfMath && dotnet build`

**Expected result:** build should generate a couple of warnings.
**Actual result:** no warnings.

If I put back the lines with `<Nullable>` in `WpfMath.csproj` and comment-out the lines in `Directory.Build.props`, then it works.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

File: Directory.Build.props (hunk 1 above)
Comment (rstm-sf) [reply Directory.Build.props:7]: Hmmm, I don’t know why it don’t evaluating in the end

Used the new api https://github.com/dotnet/msbuild/issues/5171

File: src/WpfMath/WpfMath.csproj [hunk 2]
```diff
@@ -37,9 +36,10 @@ Fixed:
         <EmbedUntrackedSources>true</EmbedUntrackedSources>
         <IncludeSymbols>true</IncludeSymbols>
         <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    </PropertyGroup>
 
-        <Nullable Condition="'$(TargetFramework)' == 'netcoreapp3.1'">enable</Nullable>
-        <Nullable Condition="'$(TargetFramework)' == 'net452'">annotations</Nullable>
+    <PropertyGroup Condition=" $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net452')) ">
+        <IncludePackageReferencesDuringMarkupCompilation>true</IncludePackageReferencesDuringMarkupCompilation>
```
Comment (rstm-sf) [inline src/WpfMath/WpfMath.csproj:42]: This is required for building without legacy SDK

**Review by ForNeVeR — APPROVED** [review]

**Comment by ForNeVeR** [comment]
Thanks for help!

**Pull request merged**